doc/library/tensor/nnet/ctc.txt

```diff
@@ -17,7 +17,7 @@
     Unfortunately, Windows platforms are not yet supported by the underlying
     library.
 
-.. module:: ctc
+.. module:: theano.ctc.root
```

Comment:
Fixed, I was editing while also editing the description of the PR, I apologize for the mistake.